Sources/PointFree/Emails/SendEmail.swift

```diff
@@ -91,6 +92,67 @@ public func prepareEmail(
   )
 }
 
+public func prepareEmailV2(
```

Comment:
Once we convert the rest of the emails to this new system we can get rid of `prepareEmail` in favor of this.